docs/Parameters.rst

```diff
@@ -28,6 +28,26 @@ By using config files, one line can only contain one parameter. You can use ``#`
 
 If one parameter appears in both command line and config file, LightGBM will use the parameter from the command line.
 
+For the Python and R packages, any parameters that accept a list of values can be specified in those languages' default array types.
```

Comment:
Do you think this info is worth to mention?
```suggestion
For the Python and R packages, any parameters that accept a list of values (usually they have ``multi-xxx`` type, e.g. ``multi-int`` or ``multi-double``) can be specified in those languages' default array types.
```

Reply:
yes that's a nice addition! Thank you.